### PR TITLE
Add panels for showing caption

### PR DESCRIPTION
**Pull request by WatanabeToshimitsu**
## What?
- Add panel for showing current caption
- Add panel for selecting scene

## Why?
- 現在時刻に合わせてキャプションを表示したいから
- シーンの選択を簡単に行いたいから

## Screenshot or video
CurrentCaptionPanel
SceneSelectPanel

### CODE REVIEW

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/webviz/pull/4#" title="Author self-approved">WatanabeToshimitsu</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fwebviz).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/dataware-tools/webviz/blob/dataware-tools/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by yusukefs — APPROVED** [review]
LGTM!

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/webviz/pull/4#" title="Author self-approved">WatanabeToshimitsu</a>*, *<a href="https://github.com/dataware-tools/webviz/pull/4#pullrequestreview-833550655" title="Approved">yusukefs</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fwebviz).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/dataware-tools/webviz/blob/dataware-tools/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
